src/tools/edit_line_tool.cpp

```diff
@@ -429,6 +429,11 @@ int EditLineTool::updateDirtyRectImpl(QRectF& rect)
 	
 	selection_extent = QRectF();
 	map()->includeSelectionRect(selection_extent);
+
+	const auto frame_extension = .001 * cur_map_widget->getMapView()
```

Comment:
I prefer to not drop the leading 0 in 0.001. (It is technically possible in C++ but uncommon in general.)

Reply:
Ack. Likely another bad habit from my C practice. :)